lib/common/xml.c

```diff
@@ -378,53 +373,50 @@ bool xml_document_dirty(xmlNode *xml)
   </change>
 </diff>
  */
-static int __xml_offset(xmlNode *xml) 
-{
-    int position = 0;
-    xmlNode *cIter = NULL;
-
-    for(cIter = xml; cIter->prev; cIter = cIter->prev) {
-        xml_private_t *p = ((xmlNode*)cIter->prev)->_private;
-
-        if (!pcmk_is_set(p->flags, xpf_skip)) {
-            position++;
-        }
-    }
-
-    return position;
-}
 
-static int __xml_offset_no_deletions(xmlNode *xml) 
+/*!
+ * \internal
+ * \brief Return ordinal position of an XML node among its siblings
+ *
+ * \param[in] xml            XML node to check
+ * \param[in] ignore_if_set  Don't count siblings with this flag set
+ *
+ * \return Ordinal position of \p xml (starting with 0)
+ */
```

Comment:
Why do some functions get a full comment block, while others just get a single line?

Reply:
No good reason. I tend to use single-line for internal functions with self-explanatory arguments and return value, just to avoid spending much time on it. Some of it is just historical -- technically doxygen blocks are necessary only for public API, so at one point that was all that had them, but it spread to internal API over the years.